os/fs/smartfs/README.txt

```diff
@@ -1,118 +1,216 @@
-SMARTFS README
+jSmartFS README
 ^^^^^^^^^^^^^^
 
 This README file contains information about the implemenation of the TinyAra
 Sector Mapped Allocation for Really Tiny (SMART) FLASH file system, SMARTFS.
 
+The same has now been upgraded to jSmartFs for TizenRT with the addition of
+2 key features - Filesystem Consistency and Two-Tier File Data Integrity.
```

Comment:
Please, change the numerical digit '2' to 'two.'

Reply:
Done